storage/interceptors/broker_public_plans_interceptor.go

```diff
@@ -82,13 +88,14 @@ func (p *publicPlanCreateInterceptor) OnTxCreate(f storage.InterceptCreateOnTxFu
 		if err != nil {
 			return nil, err
 		}
-		return newObject, resync(ctx, obj.(*types.ServiceBroker), txStorage, p.isCatalogPlanPublicFunc, p.supportedPlatforms)
+		return newObject, resync(ctx, obj.(*types.ServiceBroker), txStorage, p.isCatalogPlanPublicFunc, p.supportedPlatformsFunc, p.tenantKey)
 	}
 }
 
 type publicPlanUpdateInterceptor struct {
 	isCatalogPlanPublicFunc publicPlanProcessor
-	supportedPlatforms      func(ctx context.Context, plan *types.ServicePlan, repository storage.Repository) ([]string, error)
+	supportedPlatformsFunc  func(ctx context.Context, plan *types.ServicePlan, repository storage.Repository) (map[string]*types.Platform, error)
```

Comment:
use `supportedPlatformsProcessor`?

Reply:
should not it be similiar to isCatalogPlanPublicFunc ..?